include/bluetooth/mesh/models.rst

```diff
@@ -16,11 +19,45 @@ documentation.
 
    ../../../include/bluetooth/mesh/gen_*
 
-Common model types
-===================
+For more information about these and other models, see also `Mesh Model Overview`_.
+
+.. _bt_mesh_models_categorization:
+
+Categorization
+==============
```

Comment:
A bit confused about the levels of the headings here. Is this supposed to be on the same level as the API documentation, or is API documentation supposed to be a subtopic to Common types?
Guidelines for which characters to use for which level are here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/documentation/index.html#doc-guidelines
It works with other characters as well as long as they are used consistently. But it's a lot less confusing if we stick to the same characters in all our docs. ;) So this would mean * for L2 and = for L3.

Reply:
Right, updated.